### PR TITLE
Add model for ch.bafu.landesforstinventar-vegetationshoehenmodell_relief

### PR DESCRIPTION
**Pull request by faselm**
Test link: https://mf-chsdi3.dev.bgdi.ch/shorten/917d1a372d

Already approved

### CODE REVIEW

**Comment by b4l** [comment]
@faselm should this go into this weeks deploy?

**Comment by faselm** [comment]
@b4l Yes, I adde dthe milestone

File: chsdi/models/vector/bafu.py [hunk 1]
```diff
@@ -2357,7 +2357,18 @@ class LandesForstInventarVegetationsHoehenModell(Base, LandesForstInventarVegeta
 register(LandesForstInventarVegetationsHoehenModell.__bodId__, LandesForstInventarVegetationsHoehenModell)
 
 
-class LandesForstInventarVegetationsHoehenModellRelief(Base, LandesForstInventarVegetation, Vector):
+class LandesForstInventarVegetationRelief:
+    __tablename__ = 'vegetationshoehenmodell_relief'
+    __table_args__ = ({'schema': 'wald', 'autoload': False, 'extend_existing': True})
+    __template__ = 'templates/htmlpopup/landesforstinventar-vegetationshoehenmodell.mako'
+    __queryable_attributes__ = ['datenstand']
```
Comment (ltclm) [inline chsdi/models/vector/bafu.py:2364]: please don't use this anymore, if the attribute is defined as searchable in DIF, we are creating a Sphinx index.

Comment (ltclm) [reply chsdi/models/vector/bafu.py:2364]: see here: https://github.com/geoadmin/mf-chsdi3/pull/3691

**Review by ltclm — CHANGES_REQUESTED** [review]

File: chsdi/models/vector/bafu.py (hunk 1 above)
Comment (b4l) [reply chsdi/models/vector/bafu.py:2364]: @faselm can you remove the querryable attributes, thanks!

**Review by ltclm — APPROVED** [review]
thank you!

**Review by b4l — CHANGES_REQUESTED** [review]
I think it would be more concise to move the `__tablename__` attribute to the children and inherit from class `LandesForstInventarVegetation` instead of making a new one.

**Pull request merged**